pkgs/by-name/ba/backlight-auto/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, zig, libyuv, fetchFromGitHub }:
+stdenv.mkDerivation {
+  pname = "backlight-auto";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "lf94";
+    repo = "backlight-auto";
+    rev = "0.0.1";
```

```suggestion
    rev = version;
```

Requiring `rec` or `finalAttrs`.

---

done